pipeline/src/constructors/make_model_priors.jl

```diff
@@ -16,7 +16,7 @@ deviation 1e-1.
 
 """
 function make_model_priors(pipeline::AbstractEpiAwarePipeline)
-    transformed_process_init_prior = Normal(0.0, 0.25)
+    transformed_process_init_prior = Normal(0.0, 0.1)
```

Comment:
This reduces the initial prior not the std?

Reply:
Thats right